Dart/src/com/jetbrains/lang/dart/highlight/DartColorsAndFontsPage.java

```diff
@@ -207,13 +215,16 @@
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.getter.reference"), STATIC_GETTER_REFERENCE),
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.method.declaration"), STATIC_METHOD_DECLARATION),
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.method.reference"), STATIC_METHOD_REFERENCE),
+      new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.method.tearoff"), STATIC_METHOD_TEAR_OFF),
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.setter.declaration"), STATIC_SETTER_DECLARATION),
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.static.setter.reference"), STATIC_SETTER_REFERENCE),
 
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.top.level.function.declaration"),
                                TOP_LEVEL_FUNCTION_DECLARATION),
       new AttributesDescriptor(DartBundle.message("dart.color.settings.description.top.level.function.reference"),
                                TOP_LEVEL_FUNCTION_REFERENCE),
+      new AttributesDescriptor(DartBundle.message("dart.color.settings.description.top.level.function.reference"),
```

Comment:
Thanks to the 'Unused property' inspection, I noticed a copy/paste error here :)